### PR TITLE
[10.0] add new module account_account_tag_code

### PR DESCRIPTION
**Pull request by luc-demeyer**

### CODE REVIEW

File: account_account_tag_code/views/acccount_account_tag.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+  <data>
```
Comment (adrienpeiffer) [inline account_account_tag_code/views/acccount_account_tag.xml]: data isn't necessary

**Review by adrienpeiffer — CHANGES_REQUESTED** [review]

**Review by adrienpeiffer — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]

**Comment by gurneyalex** [comment]
@fclementic2c  you may find this one useful

**Comment by gurneyalex** [comment]
similar to #535 which can provide a similar effect with a little bit more features

**Comment by luc-demeyer** [comment]
@gurneyalex 

IMHO both PRs are complementary.

Adding a code to the account tags is different from adding a category.
The code is intended for creation of reports based upon a tax code (a mis_builder based Belgian tax declaration is under construction) whereas a category is,intended for grouping different account tags together.

We could consider though to merge both extensions to the account tags into a single module since at the end of the day both extensions facilitate the creation of financial reports.

**Review by JordiBForgeFlow — APPROVED** [review]

**Pull request closed without merging**